softdevice_controller/README.rst

```diff
@@ -43,8 +43,8 @@ Proprietary feature support:
 | QoS Conn Event Reports   | X               | X            | X         |
 +--------------------------+-----------------+--------------+-----------+
 
-.. note::
-   Please note the following when using the library for the nRF5340:
+
+Note the following when using the library for the nRF5340:
 
```

Comment:
Why not just a note as follows:

`.. note::`
    Low Latency Packet mode is not supported on the nRF53 Series.